client/flutter/lib/api/who_service.dart

```diff
@@ -33,6 +33,18 @@ class WhoService {
     return true;
   }
 
+
+  static Future<Map<String,dynamic>> getCaseStats() async {
+    Map<String, String> headers = await _getHeaders();
+    var url = '$serviceUrl/getCaseStats';
+    var response = await http.post(url, headers: headers, body: '');
```

Comment:
why did we set up this API endpoint as a `POST` instead of a `GET` request?

Reply:
Its how the present RPC framework is.  Only POST is allowed.  It was discussed in the server PR.